src/bp/ui-studio/src/web/views/Config/index.tsx

```diff
@@ -0,0 +1,507 @@
+import { Button, Callout, FileInput, FormGroup, InputGroup, Intent, ITreeNode, TextArea, Tree } from '@blueprintjs/core'
+import axios from 'axios'
+import { BotConfig } from 'botpress/sdk'
+import { BotEditSchema } from 'common/validation'
+import Joi from 'joi'
+import _ from 'lodash'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import Select from 'react-select'
+import { fetchBotInformation } from '~/actions'
+import confirmDialog from '~/components/Shared/ConfirmDialog'
+import { Container, SidePanel, SidePanelSection } from '~/components/Shared/Interface'
+import { toastFailure, toastSuccess } from '~/components/Shared/Utils/Toaster'
+
+import style from './style.scss'
+
+const statusList: SelectItem[] = [
+  { label: 'Published', value: 'public' },
+  { label: 'Collaborators Only', value: 'private' },
+  { label: 'Unmounted', value: 'disabled' }
+]
+
+const axiosConfig = {
+  baseURL: 'api/v1/'
+}
+
+interface StateBot {
+  name: string
+  status: SelectItem
+  description: string
+  selectedDefaultLang: SelectItem
+  selectedLanguages: SelectItem[]
+  website: string
+  phoneNumber: string
+  emailAddress: string
+  termsConditions: string
+  privacyPolicy: string
+  avatarUrl: string
+  coverPictureUrl: string
+}
+
+interface StateVars {
+  botId: string
+  licensing: Licensing
+  languages: SelectItem[]
+  statuses: SelectItem[]
+  error: any
+  isSaving: boolean
+  isMainOpen: boolean
+  isDetailsOpen: boolean
+  isPicturesOpen: boolean
+  nodes: ITreeNode[]
+}
+
+type State = StateBot & StateVars
+
+interface Licensing {
+  isPro: boolean
+}
+
+interface SelectItem {
+  label: string
+  value: string
+}
+
+class ConfigView extends Component<Props, State> {
+  initialFormState: StateBot = {
+    name: '',
+    status: { value: '', label: '' },
+    description: '',
+    selectedDefaultLang: { value: '', label: '' },
+    selectedLanguages: [],
+    website: '',
+    phoneNumber: '',
+    emailAddress: '',
+    termsConditions: '',
+    privacyPolicy: '',
+    avatarUrl: '',
+    coverPictureUrl: ''
+  }
+
+  sideBarNodes: ITreeNode[] = [
```

Comment:
A tree view is a bit over-complicated since we only have 3 choices for now. We have an ItemList which would fit better for a menu list (the same one we have in CMS, NLU, etc.)